store/src/db/migrations.rs

```diff
@@ -18,6 +18,7 @@ pub static MIGRATIONS: Lazy<Migrations> = Lazy::new(|| {
             notes
         (
             block_num INTEGER NOT NULL,
+            batch_index INTEGER NOT NULL,
             note_index INTEGER NOT NULL,
```

Comment:
We should probably add comments here to clarify what `batch_index` and `note_index` represent.